### ISSUE TITLE
Mic not found

### ISSUE THREAD

**Issue by BigMuscleLeo** [CyberMor/sampvoice#39]
Hi i have gotten in a bit of a sticky situation , when i first got the mod the mic was detected but i wasnt allowed to talk because of the rules on the server but i was able to listen to other players , after a few relogs and changing nothing whatsoever, i couldn't hear anyone anymore and the mic was not detected anymore. I have tried reinstalling samp, voice chat, gta sa all together, in different orders and separate and it doesn't seem to work. To me it makes no sense as it used to work so in the same circumstances but now doesnt . 
https://gyazo.com/d8531584d95dc7562cb676f89aea2ae2
Cheers.-LEO

**Comment by VitoliniSAMP** [comment]
I got this problem too

**Comment by MarlonPra** [comment]
> Hi i have gotten in a bit of a sticky situation , when i first got the mod the mic was detected but i wasnt allowed to talk because of the rules on the server but i was able to listen to other players , after a few relogs and changing nothing whatsoever, i couldn't hear anyone anymore and the mic was not detected anymore. I have tried reinstalling samp, voice chat, gta sa all together, in different orders and separate and it doesn't seem to work. To me it makes no sense as it used to work so in the same circumstances but now doesnt .
> https://gyazo.com/d8531584d95dc7562cb676f89aea2ae2
> Cheers.-LEO

Were you able to solve the problem?

**Comment by VitoliniSAMP** [comment]
> > Hi i have gotten in a bit of a sticky situation , when i first got the mod the mic was detected but i wasnt allowed to talk because of the rules on the server but i was able to listen to other players , after a few relogs and changing nothing whatsoever, i couldn't hear anyone anymore and the mic was not detected anymore. I have tried reinstalling samp, voice chat, gta sa all together, in different orders and separate and it doesn't seem to work. To me it makes no sense as it used to work so in the same circumstances but now doesnt .
> > https://gyazo.com/d8531584d95dc7562cb676f89aea2ae2
> > Cheers.-LEO
> 
> Were you able to solve the problem?

Yes, you need to upload the gamemode to a server or vps, because host of pc dont support but in server, the problem its the ports maybe

**Comment by Ghyfary17** [comment]
https://www.youtube.com/watch?v=CEzDU49XLd8 I have a video recommendation for your problem and don't forget to subscribe to my channel
